README.md

```diff
@@ -29,3 +29,31 @@ Gitpod is provided as a [managed Saas version](https://gitpod.io) with a free su
 Open any GitLab, GitHub, or Bitbucket project, by prefixing the project's URL with `gitpod.io/#`. 
 
 E.g. https://gitpod.io/#https://github.com/gitpod-io/gitpod
+
+## Documentation
+
+All documentation can be found on https://docs.gitpod.io.
+For example, see [Introduction](https://www.gitpod.io/docs/) and [Getting Started](https://www.gitpod.io/docs/getting-started/) sections. 📚
+
+## Questions
+
+For questions and support please use the [community forum](http://community.gitpod.io/).
```

Comment:
Maybe we could specifically mention gitpod.io here, too. My hope would be that we can bring down the number of issues due to service issues.

Reply:
This was also the intent behind [this proposal](https://typefox.slack.com/archives/C01APSL1RPV/p1605108490052500) (internal).